frontend-zs/src/components/stories/StoryPagination.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useEffect } from 'react';
+import { Row, Pagination, Col } from 'react-bootstrap'
+import './Story.scss'
+
+export default function StoryPagination(totalPages, onPageChanged) {
+    const [currentPage, setCurrentPage] = useState(1)
+
+    useEffect(() => {
+        goToPage(1)
+    }, [totalPages])
```

Comment:
This triggers a warning, and we can't add goToPages to the dependencies since it will change on each render (as far as I understand). There are several solutions to this problem: 

https://stackoverflow.com/questions/55840294/how-to-fix-missing-dependency-warning-when-using-useeffect-react-hook

I would personally wrap the goToPage into its own callback hook. If that doesn't work the way we want to it to work for some reason, perhaps we could disable the warning. 

 I have a question, why is totalPages a dependency for this hook? 🤔

Reply:
Also one more thing.. Do we really need this hook, since we already set the default state for current page to 1?

Reply:
`totalPages` is a dependency because the component is mounted before the data si fetched. It was the only solution that worked. This way the component is updated after the data is fetched.

Reply:
If I remove useEffect the first page will not be loaded: 

![image](https://user-images.githubusercontent.com/29261118/80283829-4c2f6100-8723-11ea-8cf3-d61374aadc5b.png)

Reply:
I have also tried it with a callback hook and it doesn't work the way we want to it to work. I can disable the warning easily with `// eslint-disable-next-line react-hooks/exhaustive-deps`

Reply:
Then let's disable the warning